apps/site/assets/ts/shuttles/__tests__/ShuttlesOverviewTest.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { mount, ReactWrapper } from "enzyme";
+import { createReactRoot } from "../../app/helpers/testUtils";
+import ShuttlesOverview from "../components/ShuttlesOverview";
+import { Route } from "../../__v3api";
+
+const route: Route = {
+  description: "",
+  direction_destinations: { 0: "Oak Grove", 1: "Forest Hills" },
+  direction_names: { 0: "Inbound", 1: "Outbound" },
+  id: "Orange",
+  name: "Orange",
+  long_name: "Orange Line",
+  type: 1
+};
+
+it("it renders", () => {
+  createReactRoot();
+  const tree = renderer
+    .create(<ShuttlesOverview places={route.direction_destinations} />)
+    .toJSON();
+  expect(tree).toBeTruthy();
```

Comment:
@thecristen let's use a "snapshot" test here. (See anyplace in the frontend tests we've called `expect(tree).toMatchSnapshot()`. The idea with a snapshot test is, we have a file checked into the repo that contains the markup that this component should render to, and the test verifies the actual markup generated against the content of this "snapshot" file.

Of course, the snapshot file doesn't exist until we create it, and thus running `npm run test:jest` will fail on this test. But you can have the snapshot automatically created by running `npm run test:jest:update`, which will run the test suite and generate/regenerate snapshots as needed.

The resulting snapshot file should be added to the PR as well.